packages/components/src/components/toggle/toggle.component.ts

```diff
@@ -185,11 +181,17 @@ class Toggle extends FormInternalsMixin(DataAriaLabelMixin(FormfieldWrapper)) im
 
     public override render() {
       return html`
-        <div class="mdc-toggle__container mdc-focus-ring">
+        <mdc-statictoggle
+          ?checked="${this.checked}"
+          ?disabled="${this.disabled}"
+          size="${this.size}"
+          class="mdc-focus-ring"
+          part="container"
+        >
           <input
             id="${this.id}"
             type="checkbox"
-            class="mdc-toggle__input"
+            part="input"
```

Comment:
```suggestion
            part="toggle-input"
```

We have observed that the part name of "input" doesn't work as expected. I'm not sure of the reason (Guess it acts as a keyword of the tagname)

Reply:
done.